egs/vais1000/tts1/RESULTS.md

```diff
@@ -0,0 +1,189 @@
+# v.0.5.0: fastspeech.v2 1024 pt window / 256 pt shift / GL 1000 iters
```

Comment:
It seems that this file is from ljspeech.
Please make a new RESULTS.md including the results of this recipe.